live/tests/unit/components/navbar-header-test.js

```diff
@@ -8,36 +8,76 @@ describe('Unit | Component | Navbar Header Component', function() {
   const sessionStubResolve = Ember.Service.extend({ isAuthenticated: true });
   const sessionStubReject = Ember.Service.extend({ isAuthenticated: false });
 
-  describe('#isUserLogged true case', function() {
-
+  context('When user is logged', function() {
     beforeEach(function() {
       this.register('service:session', sessionStubResolve);
       this.inject.service('session', { as: 'session' });
     });
 
-    it('should return true, when user is authenticated', function() {
-      // when
-      const component = this.subject();
+    describe('#isUserLogged', function() {
+      it('should return true', function() {
+        // when
+        const component = this.subject();
 
-      // then
-      expect(component.get('isUserLogged')).to.equal(true);
+        // then
+        expect(component.get('isUserLogged')).to.equal(true);
+      });
     });
 
-  });
+    describe('#menu', function() {
+      it('should only contains value for a logged user', function() {
```

Comment:
`should only contains links for a logged user`

Reply:
Plutôt: should only contains permanent menu items